ttm/publisher.py

```diff
@@ -78,7 +78,7 @@ def overall_result(self, snapshot):
                         # update reference
                         self.issues_to_ignore[ref] = build_nr
 
-                if ignored:
+                if ignored or job['result'] == 'parallel_failed':
```

Comment:
I wonder where other "uninteresting" jobs are ignored, e.g. "cancelled", etc.

But looks like it gets the job done :)

Reply:
a canceled one is not uninteresting

Reply:
Hm, so all others would count as "important" for TTM, including obsolete which should only happen when there is actually a new build so TTM would look that that one. That makes sense, yes.

Reply:
Obsolete is only on 'replaced snapshots' - since we only rnu ttm against 'latest', it's not realistically happening.

In any case:
https://github.com/openSUSE/openSUSE-release-tools/blob/5728167f14fe3e72009e7f764a8455b3c8f5a6a9/ttm/publisher.py#L52

There we check for potential failed ones to handle.

An alternative option was to just remove parallel_failed from that list - but that in turn would mean it is not listed as 'failed' anymore in the ttm output; as I still want to see it (as ignored) I opted for the change as submitted